Mathlib/Topology/Homeomorph.lean

```diff
@@ -40,9 +40,9 @@ variable {X : Type*} {Y : Type*} {Z : Type*}
 structure Homeomorph (X : Type*) (Y : Type*) [TopologicalSpace X] [TopologicalSpace Y]
     extends X ≃ Y where
   /-- The forward map of a homeomorphism is a continuous function. -/
-  continuous_toFun : Continuous toFun := by continuity
+  continuous_toFun : Continuous toFun := by dsimp; fun_prop
```

Comment:
Looking at the bench, the increase in `dsimp` time is greater than the decrease in `aesop` time. I don't know what's the right choice here. Without `dsimp`, it looks at goals like `Continuous (Equiv.mk ..).invFun` and can't proceed.